README.md

```diff
@@ -25,6 +25,12 @@ or you can directly install it for the current user by running this command:
 python setup.py install
 ```
 
+SASS is required in order to precompile the custom stylesheets:
```

Comment:
Is this still needed considering that we embed bootstrap directly? Can we use something based on libsass instead that doesn't require ruby?